test/__snapshots__/LiquidityAmounts.spec.ts.snap

```diff
@@ -1,21 +1,21 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`LiquidityAmounts #getAmount0ForLiquidity gas 1`] = `352`;
+exports[`LiquidityAmounts #getAmount0ForLiquidity gas 1`] = `479`;
```

Comment:
Any idea why this is going up? This library is used when adding liquidity so best if it didn't change too much

Reply:
I think unchecked -> checked math. let me see if I can improve

Reply:
added a few unchecked blocks and much closer now